### PR TITLE
Add Reference repo to git clone

### PR DESCRIPTION
**Pull request by AdamBrousseau**
Signed-off-by: Adam Brousseau <adam.brousseau@ca.ibm.com>

### CODE REVIEW

**Comment by AdamBrousseau** [comment]
Test build
https://openj9-jenkins.osuosl.org/job/CopyrightCheck-OpenJDK20/9

File: buildenv/jenkins/jobs/infrastructure/copyrightCheckDir.groovy [hunk 1]
```diff
@@ -40,7 +40,17 @@ timeout(time: 6, unit: 'HOURS') {
...
                         ROOTDIR="ROOTDIR=${params.rootDir}"
                     }
-                    checkout scm
+                    checkout changelog: false, poll: false,
+                            scm: [$class: 'GitSCM',
+                                branches: [[name: scm.branches[0].name]],
+                                doGenerateSubmoduleConfigurations: false,
+                                extensions: [[$class: 'CloneOption',
+                                                depth: 0,
+                                                noTags: false,
```
Comment (pshipton) [inline buildenv/jenkins/jobs/infrastructure/copyrightCheckDir.groovy:49]: this could be true?

**Comment by pshipton** [comment]
Pls update the copyright.

**Comment by pshipton** [comment]
The clone doesn't show anything about a reference, how do we know it's working?
```
17:03:21  Checking out Revision 9b4fdac5ce0038071762f7bf40b12e70962dfb9b (refs/remotes/origin/copyright_refrepo)
17:03:21   > git config remote.origin.url https://github.com/adambrousseau/openj9-openjdk-jdk.git # timeout=10
17:03:21   > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
17:03:21   > git rev-parse refs/remotes/origin/copyright_refrepo^{commit} # timeout=10
17:03:21   > git config core.sparsecheckout # timeout=10
17:03:21   > git checkout -f 9b4fdac5ce0038071762f7bf40b12e70962dfb9b # timeout=10
17:05:58  Commit message: "Add Reference repo to git clone"

```

**Comment by pshipton** [comment]
I guess I need to look earlier in the output
```
17:02:36  Using reference repository: /home/jenkins/openjdk_cache
```

File: buildenv/jenkins/jobs/infrastructure/copyrightCheckDir.groovy (hunk 1 above)
Comment (pshipton) [reply buildenv/jenkins/jobs/infrastructure/copyrightCheckDir.groovy:49]: @AdamBrousseau ?

Comment (AdamBrousseau) [reply buildenv/jenkins/jobs/infrastructure/copyrightCheckDir.groovy]: updated

**Review by pshipton — APPROVED** [review]

**Pull request merged**